src/app/frontend/about/style.scss

```diff
@@ -18,5 +18,6 @@
 .kd-about-logo-icon {
   @include kd-icon-size(8 * $baseline-grid);
 
-  margin: 0 (2 * $baseline-grid) 0 0;
+  margin: 0px;
```

Comment:
`0` doesn't need unit.

Reply:
I'd like also to know why it was changed.

Reply:
The previous code always changes the right margin, but with this current code, it will change the margin of the end side of the element, which in LTR languages is the right side (like before) and in RTL languages is the left side.